src/rest/Facet/FacetSearchResponse.ts

```diff
@@ -1,18 +1,32 @@
-/**
- * TODO: documentation
- */
 export interface IFacetSearchResultValue {
+  /**
+   * The custom facet value display name, as specified in the `captions` argument of the facet request.
```

Comment:
When we expose the `captions` attribute of the IFacetRequest model in the public reference documentation, we will want to link to it here.

We can't do anything yet, though.